client/src/pages/Maintenance/Maintenance.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../scssStyles/mixins.scss';
+
+.full-page {
+  width: 100000vh;
+  height: 100000vh;
+
+  background: linear-gradient(
+      180deg,
+      rgba(255, 255, 255, 0) 0%,
+      rgba(221, 204, 231, 0.21875) 38.02%,
+      #621790 100%
+    ),
+    #935ebf;
+}
+
+.box {
+  position: fixed;
+  top: 50%;
```

Comment:
Line 18 to 22 might not be needed
`display: flex` should handle the centering of content with
```
justify-content: center; align-items: center;
```
what you used below

Reply:
i'm not sure why, but it doesn't seem to work 😅

Reply:
Try removing `position: fixed`

Reply:
Check my latest commit